x-pack/plugins/security_solution/scripts/endpoint/common/security_user_services.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Client } from '@elastic/elasticsearch';
+import { userInfo } from 'os';
+
+export const createSecuritySuperuser = async (
```

Comment:
Not needed now, but we could use this in our generator script to create users with superuser roles